messaging/src/main/java/org/axonframework/eventhandling/StreamingEventProcessor.java

```diff
@@ -91,6 +91,22 @@ public interface StreamingEventProcessor extends EventProcessor {
      */
     CompletableFuture<Boolean> splitSegment(int segmentId);
 
+    /**
```

Comment:
**(Super nit incoming...)** This is super OCD of me, but why isn't this method placed right after the `releaseSegment` method? :sweat_smile: 
I would see release and claim as counterparts of one another, just as split and merge are counterparts of one another.

Reply:
I adjusted the ordering for your pleasure :)